typescript/__tests__/testProgramaticallyCreateConfig.ts

```diff
@@ -0,0 +1,20 @@
+import { AIConfigRuntime } from "../lib/config";
+import { InferenceSettings } from "../types";
+
+describe("test extract_override_settings", () => {
+  // shared state
+  let aiConfigRuntime: AIConfigRuntime = AIConfigRuntime.create(
+    "Untitled AIConfig",
+    undefined,
+    undefined,
+    { models: { testmodel: { topP: 0.9 } } }
+  );
+
+  it("Test Case 1: Retrieve global setting from AIConfig with 1 model", () => {
```

Comment:
nit: change from `it` to `test`. They are the same.

Also update the description:
`test("Retrieving global settings from AIConfig with 1 model")`.
Reson: in Jest the test names are supposed to read like a proper sentence. In this case: "test retrieving global settings from AIConfig with 1 model"

If you use `it` is can be `it("Retrieves global settings from AIConfig with 1 model")` so the sentence reads "it retrieves global settings from AIConfig with 1 model"

Reply:
that's a cool style guideline. Is there a javascript/typescript style guidelines like Python's PEP?